gulpfile.js

```diff
@@ -67,6 +69,16 @@ gulp.task('lib', function() {
 gulp.task('flow', function() {
   return gulp
     .src(paths.lib.src)
+    .pipe(gulpModuleMap(moduleMapOpts))
+    .pipe(gulpStripProvidesModule())
```

Comment:
I'm pretty sure we still need the providesModule bit for the projects that are using Haste (eg Relay) - see #98. We should do the same test as I did there.